README.md

```diff
@@ -5,3 +5,62 @@ HTTP Accept-* headers parsing.
 [![Build Status](https://secure.travis-ci.org/hapijs/accept.png)](http://travis-ci.org/hapijs/accept)
 
 Lead Maintainer - [Eran Hammer](https://github.com/hueniverse)
+
+## Introduction
+
+Accept helps to answer the question of whether the browser can use certain content encoding techniques, such as gzip or deflate.  Accept will parse the headers of a HTTP request and tell you what the preferred encoding is, or if given a list of possible encodings, it will respond with the first available encoding that matches something in the list.
+
+Additional details about Accept headers and content negotiation can be found in [IETF RFC 7231, Section 5.3](https://tools.ietf.org/html/rfc7231#section-5.3).
+
+## Parsing for a specific encoding
+
+Using the `encoding` function will return a `string` matching a type of encoding.  `Encoding` takes two parameters, `encoding(encodingHeader, [preferences])`.  The headers parameter is required, but the preferences are optional.  If no preferences are provided, encoding will find the first supported encoding option and return that.
```

Comment:
Need to be consistent. Functions should have `encode()` parenthesis and pay attention to case (second encoding is upper case).